commands/config/get.js

```diff
@@ -63,7 +63,7 @@ var Get = Command.extend({ use: ['session', 'auth'],
         '    --app-name\n'.yellow +
         '    --default-size'.yellow + ' (the default number of hits returned for the current session)\n'.grey+
         '    --range-separator'.yellow + ' (used to separate two datetimes when specifying time range)\n'.grey+
-        '    --region'.yellow + ' (which Sematext AWS region to use [US|EU])\n'.grey+
+        '    --region'.yellow + ' (which Sematext AWS region is used [US|EU])\n'.grey+
```

Comment:
I wouldn't call this AWS region. Neither "US" or "EU" are really AWS regions.  Just regions. :)

Reply:
Makes sense :)